### ISSUE TITLE
Shortened DOIs for hashtags?

### ISSUE THREAD

**Issue by hammer** [idoerg/papertooter#1]
As discussed on Mastodon, I'm curious if it might be better to use shortened DOIs, either from https://shortdoi.org or simply by eliding rather than replacing non-alphanumeric characters.

**Comment by idoerg** [comment]
@hammer I agree, papertooter uses short DOIs now as default. Long DOIs can still be used. It's a policy call, basically.

Note that at the moment the documentation is not keeping up with the features. Give me a couple of days. :)
